mmv1/products/certificatemanagerpublicca/ExternalAccountKey.yaml

```diff
@@ -0,0 +1,70 @@
+# Copyright 2023 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Api::Resource
+name: 'ExternalAccountKey'
+base_url: 'projects/{{project}}/locations/global/externalAccountKeys'
```

Comment:
We may want to add a `location` `url_param_only` field that's either required or has a default of global. That'll simplify supporting other locations in the future.

I'd lean towards a required field since it'll save some headaches later, and most users will be copy-pasting existing configs so it won't require typing. That may confuse some users who think they can supply a different value though, so if it will be a while making it non-configurable or setting a default may be preferable.